app/Http/Controllers/Api/V1/ActivityController.php

```diff
@@ -627,14 +632,25 @@ public function getH5pResourceSettingsShared(Activity $activity)
     private function hasPermission(Activity $activity)
```

Comment:
Add the comments at the top of the function like parameters receiving, return type data etc... Like we've for other functions.